core/jazz_ui/e2e/page-objects/jazzservices.po.ts

```diff
@@ -108,5 +109,11 @@ export class Jazz {
   getAssetStatusVerify(){
     return element(by.xpath('//li[@class="x caps active"]'));
   }
+  getLamdaName(){
+    return element(by.xpath('//div[@class="table-row pointer"]/div[text()="function"]/preceding-sibling::div'));
```

Comment:
is this guaranteed?
this is more like a hacky way to capturing the name

Reply:
yes Sid Its guaranteed since we are taking the Lamda name with the text "function" under the corresponding table names